specification/security/resource-manager/Microsoft.Security/stable/2023-11-15/apiCollections.json

```diff
@@ -0,0 +1,473 @@
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the provisioning state of the API collection.",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "InProgress"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Succeeded"
+              },
+              {
+                "value": "Failed"
+              },
+              {
+                "value": "Canceled"
+              },
+              {
+                "value": "InProgress"
+              }
+            ]
+          }
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the API collection."
+        },
+        "discoveredVia": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource Id of the resource from where this API collection was discovered.",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.ApiManagement/service"
+              }
+            ]
+          }
+        },
+        "baseUrl": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The base URL for this API collection. All endpoints of this API collection extend this base URL."
+        },
+        "numberOfApiEndpoints": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of API endpoints discovered in this API collection."
+        },
+        "numberOfInactiveApiEndpoints": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of API endpoints in this API collection that have not received any API traffic in the last 30 days."
+        },
+        "numberOfUnauthenticatedApiEndpoints": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of API endpoints in this API collection that are unauthenticated."
+        },
+        "numberOfExternalApiEndpoints": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of API endpoints in this API collection for which API traffic from the internet was observed."
+        },
+        "numberOfApiEndpointsWithSensitiveDataExposed": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of API endpoints in this API collection which are exposing sensitive data in their requests and/or responses."
+        },
+        "sensitivityLabel": {
```

Comment:
>sensitivityLabel [](http://example.com/codeflow?start=9&length=16)

Does this string take a fixed number of possible values? If so, best to model it as string enum.

Reply:
Unfortunately, it's a freeform string because the customer can specify any value. So it's impossible to model this as an enum. Here's a discussion I had for the same comment which has more details: https://github.com/Azure/azure-rest-api-specs-pr/pull/14134#discussion_r1298847440